clients/privacy-center/common/hooks/useCookie.ts

```diff
@@ -0,0 +1,49 @@
+import { getCookie, setCookie, Types } from "typescript-cookie";
+
+const FIDES_USER_DEVICE_ID_COOKIE_NAME = "fides_user_device_id";
+const MAX_AGE_DAYS = 1;
+
+const CODEC: Types.CookieCodecConfig<string, string> = {
+  decodeName: decodeURIComponent,
+  decodeValue: decodeURIComponent,
+  encodeName: encodeURIComponent,
+  encodeValue: encodeURIComponent,
+};
+
+export const setFidesUserDeviceIdCookie = () => {
+  // Generates a new `fides_user_device_id` and stores it to the cookie
+  if (typeof document === "undefined") {
```

Comment:
a docstring in this format helps IDEs tell you what a func is about 

```suggestion
/**
 * Generates a new `fides_user_device_id` and stores it to the cookie
 */
export const setFidesUserDeviceIdCookie = () => {
  if (typeof document === "undefined") {
```